### DISCUSSION TITLE
Current User fields are not updated

### DISCUSSION THREAD

**Discussion by juangbm** [Budibase/budibase#10395] [Help] [answered]
Hi,

I have added some fields to the Users database, but when I update those fields and are correct in the database, I cannot get the new values when using {{ Current User.xxxx }}
After updating I Refresh the Data provider, but when I try to get the values from another screen, I still get the previous ones.
Any idea about how to get the new values?

Thanks!

**Comment by ConorWebb96** [comment]
Hey @juangbm,

Could we get a bit more information on this? When you say you've added fields to the users database, do you mean the users table within your app? 

Would you also provide some screenshots of the issues you are experiencing? Just to help us with understanding the issue.

Thanks in advance!

**Reply by juangbm** [reply]
Hi @ConorWebb96 

This is the Users table from my app:

![image](https://user-images.githubusercontent.com/22911293/233982204-cb78e7ef-5ebd-4aa1-8f5c-ffbff34a68ce.png)

I have a screen where I display info related to the fields COUNTRY and COUNTRY_INTID from the Users table:
![image](https://user-images.githubusercontent.com/22911293/233984331-1a1e15d5-cfec-447e-a46e-c6114eefc660.png)

If I clicked on Change Country, I open a modal where the user can change his country (updating the Users table), values appear correctly in the database after selecting the new country, but reloading the previous screen does not reflect those changes...

Thanks

**Reply by ConorWebb96** [reply, marked as answer]
Hey @juangbm,

I've replicated what you have shown and have created an issue related to it here: https://github.com/Budibase/budibase/issues/10397 
Feel free to add additional information if you think it would be helpful in the comments!

Thank you for raising this!
